redirects/vets-gov-to-va-gov.yml

```diff
@@ -625,3 +625,6 @@
 - vets_gov_src: welcome-to-va/
   va_gov_dest: welcome-kit/
   retain_path: false
+- vets_gov_src: /facilities/
+  va_gov_dest: /find-locations/
+  retain_path: true
```

Comment:
Ah, this is because I originally forgot the `aliases` property on the Facility Locator before we collected the data 😞 Thanks for grabbing this.